config/locales/en.yml

```diff
@@ -902,6 +903,18 @@ en:
         reload: Reload summary
         with_extra_charge: 'with extra charge:'
         without_extra_charge: 'without extra charge:'
+    bank_accounts:
+      index:
+        account_statement: Account statement
+        title: Bank Accounts
+    bank_transactions:
+      index:
+        search_placeholder: Search ...
```

Comment:
What about moving this to a general i18n section (like cancel etc.)?

Reply:
agree